docs/import-data/csv/csv-import-tool.md

```diff
@@ -44,8 +45,9 @@ the complexity of your data.
 
 Here are two examples of how to use the CSV import tool depending on
 the complexity of your data:
-- [One type of nodes and relationships](#one-type-of-nodes-and-relationships)
-- [Multiple types of nodes and
+- [Examples](#examples)
+  - [One type of nodes and relationships](#one-type-of-nodes-and-relationships)
+  - [Multiple types of nodes and relationships](#multiple-types-of-nodes-and-relationships)
   relationships](#multiple-types-of-nodes-and-relationships)
 
 ### One type of nodes and relationships
```

Comment:
Where is the csv import tool on the side tab?

Reply:
As i remember correctly we are no longer instructing people to use it?
@kgolubic